src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -1534,6 +1534,25 @@ public function validateProhibitedUnless($attribute, $value, $parameters)
         return true;
     }
 
+    /**
+     * Validate that other attributes do not exist when this attribute exists.
+     *
+     * @param  string  $attribute
+     * @param  mixed  $value
+     * @param  mixed  $parameters
+     * @return bool
+     */
+    public function validateProhibits($attribute, $value, $parameters)
+    {
+        foreach ($parameters as $other) {
+            if (Arr::has($this->data, $other)) {
+                return false;
+            }
+        }
+
+        return true;
```

Comment:
```suggestion
        return !Arr::hasAny($this->data, $parameters);
```

Documentation for `Arr::hasAny()`:
https://laravel.com/docs/8.x/helpers#method-array-hasany